zenoh-ext/src/querying_subscriber.rs

```diff
@@ -913,3 +901,34 @@ where
         std::future::ready(self.res_sync())
     }
 }
+
+fn register_handler(
+    state: Arc<Mutex<InnerState>>,
+    callback: Arc<dyn Fn(Sample) + Send + Sync>,
+) -> RepliesHandler {
+    zlock!(state).pending_fetches += 1;
+    // pending fetches will be decremented in RepliesHandler drop()
+    RepliesHandler { state, callback }
+}
+
+fn run_fetch<
+    Fetch: FnOnce(Box<dyn Fn(TryIntoSample) + Send + Sync>) -> ZResult<()>,
+    TryIntoSample,
+>(
+    fetch: Fetch,
+    handler: RepliesHandler,
+) -> ZResult<()>
+where
+    TryIntoSample: TryInto<Sample>,
+    <TryIntoSample as TryInto<Sample>>::Error: Into<zenoh_core::Error>,
+{
+    log::debug!("Fetch");
```

Comment:
I would make the debug message more exhaustive.

Reply:
Done